Prox/Prox/RemoteConfigKeys.swift

```diff
@@ -36,7 +36,7 @@ class RemoteConfigKeys {
 
     // The search radius the app will use to query the Firebase database.
     // This is measured in kilometers. The default value is 4 km.
```

Comment:
Nit: would also put a comment for `place_categories_to_hide_csv` to see long default in the plist.